### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: core/src/main/resources/hudson/views/Messages_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# The MIT License
+#
+# Copyright (c) 2013, Chunghwa Telecom Co., Ltd., Pei-Tang Huang
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/views/Messages_zh_CN.properties]: I am not sure the "year" is correct here

Comment (LinuxSuRen) [reply core/src/main/resources/hudson/views/Messages_zh_CN.properties]: I already change license content, tks

**Review by daniel-beck — APPROVED** [review]

File: core/src/main/resources/hudson/views/Messages_zh_CN.properties (hunk 1 above)
Comment (LinuxSuRen) [reply core/src/main/resources/hudson/views/Messages_zh_CN.properties]: any question?

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
@LinuxSuRen Thanks!

**Comment by LinuxSuRen** [comment]
anytime

**Pull request merged**